Automattic/vip
# Fix duplicate format option on slowlogs command

## Description

The `slowlogs` command currently prints two `--format` options:
```
  Usage: vip slowlogs [options]

  Options:
    -a, --app             Specify the app
    -d, --debug           Activate debug output
    -e, --env             Specify the environment
    -f, --format [value]  Format results (defaults to "table")
    -F, --format [value]  Output the log lines in CSV or JSON format (defaults to "text")
    -h, --help            Output the help for the (sub)command
    -l, --limit <n>       The maximum number of log lines (defaults to 500)
    -v, --version         Output the version number
```
and this is because the `format: true` option is provided in that commands setup while also providing a "custom" `format` option. It also shows `text` as a default when that's not a valid option at all.

## Changelog Description

Updated the "logs" and "slowlogs" commands to use the "table" output format by default, rather than the non-existent "text" format. This makes the help text for these commands less confusing, and makes them more consistent with the rest of the CLI.

## Pull request checklist


## New release checklist


## Steps to Test

Outline the steps to test and verify the PR here.

Example:

1. Check out PR.
1. Run `npm run build`
1. Run `node ./dist/bin/vip-slowlogs.js @{app}.{env} --help`
1. Verify only one format option with -f as a shorthand
